.changeset/pre.json

```diff
@@ -1,6 +1,6 @@
 {
   "mode": "pre",
-  "tag": "rc",
+  "tag": "exit",
```

Comment:
Are we going to use `exit` branch for release candidate now?

Reply:
`exit` means we are exiting pre-prelease mode and doing a release.